qemu/tests/cfg/watchdog.cfg

```diff
@@ -61,3 +61,8 @@
             plug_watchdog_device = i6300esb
             watchdog_action = pause
             test_type = hotplug_unplug_watchdog_device
+       - stop_cont_test:
+            test_type = stop_cont_test
+            trigger_cmd = python -c "open('/dev/watchdog', 'w').close()"
```

Comment:
Please update `python` command refer to `efad97ff`

Reply:
No error now.

Reply:
Please adopt python3, like https://github.com/autotest/tp-qemu/pull/1458/files.

Reply:
@vivianQizhu Updated.